app/proposals_whitelisting.go

```diff
@@ -86,6 +88,11 @@ var WhitelistedParams = map[ccvgov.ParamChangeKey]struct{}{
 	{MsgType: stakingtypes.ModuleName, Key: string(stakingtypes.KeyMaxEntries)}:        {},
 	{MsgType: stakingtypes.ModuleName, Key: string(stakingtypes.KeyHistoricalEntries)}: {},
 	{MsgType: stakingtypes.ModuleName, Key: string(stakingtypes.KeyBondDenom)}:         {},
+	//autopilot
```

Comment:
@strbrian just checking - did you go through all of our modules and make sure all of the param change proposals are included here?

Reply:
yes, I have gone through all of our modules and no missing params.
But there are still some missing params for sdk and ics consumer and I added them to the whitelist additionally.

Reply:
Thanks! We'll just have to remember to always add new params here going forward